android/templates/build.gradle

```diff
@@ -60,7 +60,7 @@ android {
 
 dependencies {
     implementation "net.java.dev.jna:jna:5.7.0@aar"
-    implementation files('libs/rustls-platform-verifier.jar')
+    implementation "com.nordsec.rustls:rustls-platform-verifier:0.1.1"
```

Comment:
Actually no, rustls-platform-verifer uses different versioning for its AAR package. This is the version shipped with 0.3.4 of rust crate

Reply:
In that case please add a comment above that line explaining that so that it's clear that this in not a mistake, but a deliberate choice due the the way rustls-platform-verifier is released.